app/static/tests/unit/directives/tooltip.test.ts

```diff
@@ -18,11 +42,83 @@ describe("tooltip directive", () => {
         const wrapper = mountTemplate();
         const div = wrapper.find("div");
 
-        // The custom directive sets "title" attribute, which bootstrap converts to data-bs-original-title and
-        // aria-label via the Tooltip object
-        expect(div.attributes("data-bs-original-title")).toBe("hello");
-        expect(div.attributes("aria-label")).toBe("hello");
-        expect(div.attributes("data-bs-toggle")).toBe("tooltip");
+        // checking tooltip props directly
+        expect((Tooltip.getInstance(div.element) as any)._config.placement).toBe("top");
```

Comment:
Feels like this repeated test pattern could be made neater with a little helper e.g. 
```
const expectTooltipConfig = (el: HTMLElement, title: string, placement: string, toggle: string, customClass: string) => {
  const tooltipConfig = (Tooltip.getInstance(el) as any)._config;
  expect(tooltipConfig.title).toBe(title);
...etc..
}; 
```

Reply:
done